doc/examples/ridges/plot_ridge_filter.py

```diff
@@ -0,0 +1,76 @@
+"""
+===============
+Ridge operators
+===============
+
+Ridge filters can be used to detect continuous edges, such as neurites
+_[1], tubes _[2], vessels _[3], wrinkles _[4], rivers, and other rigdge-like
+structures. The present class of ridge filters relies on the eigenvalues of
+the Hessian matrix of image intensities to detect ridge structures where the
+intensity changes perpendicular but not along the structure.
+
```

Comment:
Unfortunately, I can't pin down the practical differences. The filters kind of have different "weightings" of how much certain eigenvalues along/perpendicular to the structure are taken into account. But can't really make any statements if one is generally better for a certain structure that the others. (But this would definitely be nice to have!)